build.gradle

```diff
@@ -42,7 +42,7 @@ subprojects {
             instructionReplace "Bundle-Vendor", "Reactive Streams SIG"
             instructionReplace "Bundle-Description", "Reactive Streams API"
             instructionReplace "Bundle-DocURL", "http://reactive-streams.org"
-            instructionReplace "Bundle-Version", "1.0.1"
+            instructionReplace "Bundle-Version", "1.0.2-RC1"
```

Comment:
Perhaps `$version`?

Reply:
`instructionReplace "Bundle-Version", $version` ?

Reply:
I think String interpolation should work here (but I can be never sure with Gradle):

`instructionReplace "Bundle-Version", "$version"`

Reply:
For some awkward reason "$version" will generate "1.0.1.RC1" iso "1.0.1-RC1" :S